localization/english/dvg_government_l_english.yml

```diff
@@ -21,6 +21,9 @@
  gov_guli_confederation_desc: "A [concept_government] that consists of a confederation of sinicized Hongshan under a [Concept('concept_ruler','$RULER_TITLE_ELDER$')], selected amongst the Yaluo aligned tribes to serve as their representative when necessary"
  gov_red_sultanate: "Red Sultanate"
  gov_red_sultanate_desc: "The Sultanate of Zhusha has been in existence for hundreds of years, and with time, the Sultans came to be known as the Red Sultans."
+ gov_hereditary_dictatorship: "Hereditary Dictatorship"
+ gov_hereditary_dictatorship_desc: "A government that is a republic in name only. In practice, the country favors nepotism and leadership has a heredity transfer of power."
+ gov_red_sultanate_desc: "A nation that is a republic in name only. Transfer of power is kept within the same family much like a monarchy."
```

Comment:
This key is duplicate and seems to be incorrect?

Reply:
Incorrect how?

Reply:
oops yea, extra red_sultanate_desc shouldn't be there